test/unit/driver-specs.js

```diff
@@ -27,6 +27,42 @@ describe('driver commands', () => {
     });
   });
 
+  describe('status caching', () => {
+    let d;
+    let sandbox;
+    let jwproxyCommandSpy;
+    let jwProxyRequestsCountSpy;
+
+    beforeEach(() => {
+      d = new XCUITestDriver();
+      sandbox = sinon.sandbox.create();
+      jwproxyCommandSpy = sandbox.stub(d.wda.jwproxy, "command", async () => {
```

Comment:
Does this run? At this point `d.wda` is unintialized, I think. You probably need something like
```js
d.wda = new WebDriverAgent({major: 8, minor: 2}, {});
d.wda.setupProxy(42);
```